subworkflows/local/prepare_vcfs_test/main.nf

```diff
@@ -11,7 +11,7 @@ include { FIX_VCF_PREFIX               } from '../../../modules/local/custom/fix
 include { PUBLISH_PROCESSED_VCF        } from '../../../modules/local/custom/publish_processed_vcf'
 include { BCFTOOLS_VIEW as BCFTOOLS_VIEW_CONTIGS      } from '../../../modules/nf-core/bcftools/view'
 include { BCFTOOLS_NORM as BCFTOOLS_SPLIT_MULTI       } from '../../../modules/nf-core/bcftools/norm'
-include { BCFTOOLS_REHEADER as BCFTOOLS_REHEADER_QUERY} from '../../../modules/nf-core/bcftools/reheader'
+include { BCFTOOLS_REHEADER as BCFTOOLS_REHEADER_QUERY} from '../../../modules/local/bcftools/reheader'
```

Comment:
why was that moved to local? I usually see local -> nf-core, but rarely the other way around. Do you want to modify? If yes, than maybe a [patch](https://nf-co.re/docs/nf-core-tools/modules/patch) would be sufficient?

Reply:
I needed to modify the module itself (BCFTOOLS_REHEADER) to make it work on HPC systems. that is why I copied the tool to local and change along the way I need :D

Reply:
You think patch work for my case? I will check it out